task.sh

```diff
@@ -1 +1,22 @@
-#! /bin/bash
+#!/bin/bash
+
+# Read database credentials from environment variables
+DB_USER=${DB_USER}
```

Comment:
Using `${DB_USER}` is unnecessary here. You can simply use `$DB_USER` to reference the environment variable. The same applies to the `DB_PASSWORD` on line 5.